docs/events/content/select_content.md

```diff
@@ -6,6 +6,11 @@ title: Select Content
 
 Fire whenever a user clicks on a content link found in a list that forwards the user to the content page. List types include card collections, search results, carousels, and other similar components. 
 
+This event is to be used to understand how content is being engaged with in regard to the lists they are a part of. If you are looking to report on how much a piece of content was engaged with and want/need to know the list information, you must use "select_content"  in your GA4 report as your event and NOT "[view_content](../../events/content/view_content.md)". This is because "[view_content](../../events/content/view_content.md)" will not have associated list information.
```

Comment:
good callout, but the audience for the spec is developers, not site owners/analysts. this is better in an SDR or something imo. I do like the idea of giving developers context around what type of reporting an event enables